src/core/Table/TableRowMemoized.tsx

```diff
@@ -43,6 +44,8 @@ const TableRow = <T extends Record<string, unknown>>(props: {
     rootMargin: `${intersectionMargin}px`,
   });
 
+  const expandedHeight = React.useRef(0);
```

Comment:
I guess I would have had here just `expandedContentRef `and then take its height.. In case we need that ref for anythin else, would not need to change it.

Reply:
I think element ref's height would not stay static, would it? Since we are changing inline height on the same element